site/docs/reference/Connectors/materialization-connectors/amazon-s3-parquet.md

```diff
@@ -0,0 +1,120 @@
+
+# Apache Parquet Files in Amazon S3
+
+This connector materializes [delta updates](../../../concepts/materialization.md#delta-updates) of
+Flow collections into an S3 bucket in the Apache Parquet format.
+
+The delta updates are batched within Flow, converted to Parquet files, and then pushed to the S3
+bucket at a time interval that you set.
+
+[`ghcr.io/estuary/materialize-s3-parquet:dev`](https://ghcr.io/estuary/materialize-s3-parquet:dev)
+provides the latest connector image. You can also follow the link in your browser to see past image
+versions.
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* An S3 bucket to write files to. See [this
+  guide](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) for
+  instructions on setting up a new S3 bucket.
+* An AWS root or IAM user with access to the S3 bucket. For this user, you'll need the **access
+  key** and **secret access key**. See the [AWS
+  blog](https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/) for help finding these
+  credentials.
```

Comment:
Can we specify exactly what permissions are required?

Reply:
Updated to specify the `s3:PutObject` permission.